web/app/datasets/external/sigmax.py

```diff
@@ -97,16 +98,29 @@ def _generate_stuf_message(signal):
     """
     Generate the XML needed for Sigmax.
     """
+    logger.debug('Openbare ruimte naam in signal: "{}"'.format(
+        signal['location']['address']['openbare_ruimte']))
+    # TODO: "huisnummer toevoeging" seems missing - can be a real problem - check this
+    # TODO: check X and Y (are they WGS84, are they lon lat or lat lon?)
+    # TODO: figure out which is which with regards to the dates and times
```

Comment:
Are these functional issues registered in Jira or Github issues?

Reply:
These were reminders.I have added a JIRA task SIG220 to capture these questions and removed the TODOs.

Reply:
(There is still one TODO regarding the routing which remains. There is a JIRA task for it already.)